libsplinter/Cargo.toml

```diff
@@ -63,6 +63,7 @@ transact = { version = "0.2", features = ["sawtooth-compat"] }
 url = "1.7.1"
```

Comment:
Typo in commit message enviroments/environments

Reply:
Fixed